scene/resources/animation.h

```diff
@@ -497,6 +513,17 @@ class Animation : public Resource {
 
 	void track_get_key_indices_in_range(int p_track, double p_time, double p_delta, List<int> *p_indices, Animation::LoopedFlag p_looped_flag = Animation::LOOPED_FLAG_NONE) const;
 
+	void add_marker(double p_time, const StringName &p_name);
```

Comment:
```suggestion
	void add_marker(const StringName &p_name, double p_time);
```
For consistency with other APIs. Basically, the name comes first.